CODE_OF_CONDUCT.md

```diff
@@ -1,81 +1,105 @@
 # PureScript Community Code of Conduct
 
-The PureScript community follows some simple ground rules to protect safe, inclusive spaces to discuss the PureScript programming language.
+Our community is dedicated to helping people learn and use the PureScript programming language. Our members can't do this if they are being harassed, stalked, or discriminated against. It's also hard to do that if you're regularly made to experience emotionally draining situations or made to wonder whether you belong. Accordingly, all PureScript community members are expected to show respect and courtesy towards each other in all interactions in our community.
 
-All community members are expected to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+We need a common understanding of what it means to "show respect and courtesy towards each other" and so we have adopted the following code of conduct. This code of conduct is enforced by our moderators and applies in all community spaces.
 
-## What's our purpose?
+## Our Purpose
 
-The PureScript community channels are places to discuss the PureScript programming language. 
+The PureScript community spaces are places to discuss the PureScript programming language. In these spaces:
 
 * We talk about the language, its related libraries and infrastructure, and our experiences using it.
 * We help PureScript programmers learn the language, and we ask for help becoming stronger programmers ourselves. 
 * We discuss general functional programming concepts and other parts of programming, type theory, and mathematics that are relevant to Purescript.
 * We organize open-source work, meetups, hackathons, and other community events.
 * We discuss what we can do to grow an inclusive, friendly, respectful community.
 
-## What is okay?
+## Unacceptable Behavior
 
-Examples of behavior that contributes to a positive environment for our community include:
+The following types of behavior are unacceptable in the PureScript community and constitute violations of this code of conduct.
 
-* Demonstrating empathy and kindness toward other people.
-* Being respectful of differing opinions, viewpoints, and experiences.
-* Giving and gracefully accepting constructive feedback.
-* Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience.
+This code of conduct is a set of guidelines that helps our community protect safe, inclusive spaces to discuss PureScript. However, it doesn't enumerate all possible forms of unwelcome behavior, and incidents are evaluated on a case-by-case basis so as to understand their full context.
```

Comment:
The line above says, "The following types of behavior are unacceptable..."

And now this line says "... it doesn't enumerate all possible forms of unwelcome behavior.."

The first line originally had a list that was deleted, so was the first line supposed to be deleted as well?

Reply:
I could do a better job merging them together. The point I'm trying to make is that this is a set of guidelines, but it is ultimately up to the discretion of moderators to decide what is acceptable even if it's not technically written out here. But perhaps this phrasing is verbose and unnecessary.

Reply:
On a second look, I could just swap the two paragraphs